.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Feature Request
+about: Create a feature request
+title: Feature Proposal
+labels: enhancement
+assignees: ''
+
+---
+
+<!--- Provide a general summary of the feature request or improvement in the Title above -->
+<!--- Look through existing open and closed feature proposals to see if someone has asked for the feature before -->
+
+## Expected Behavior
+<!--- Tell us how it should work -->
+
+## Difference from Current Behavior
+<!--- Explain the difference from current behavior -->
+
+## Possible Implementation
```

Comment:
Also add a section includes adding new model / changing existing models etc?
also Migration Plan and Compatibility ?
also potential Rejected Alternatives ?

Reply:
All of those questions are really valid, however we could overwhelm a bit our users if we ask all that. I see the feature request issue as a simple "idea gathering" tool. 

I think these are questions for a design document, something we would probably would need to drive. Let me add them into a section of the contributing guide.